### PR TITLE
feat: allow usage of -q or --quiet

### PR DESCRIPTION
**Pull request by aviadhahami**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules

#### What does this PR do?
We would like to allow the usage of `-q` (== `--quiet`) as a cli flag in order to allow users to ignore warnings; A classic example will be the print-deps warning for large graphs/trees;

### CODE REVIEW

File: src/lib/print-deps.ts [hunk 1]
```diff
@@ -21,15 +21,19 @@ export async function maybePrintDepGraph(
...
+        }
         // TODO @boost: add as output graphviz 'dot' file to visualize?
         console.log(JSON.stringify(depGraph.toJSON(), null, 2));
       } else {
-        console.log(
-          '--print-deps option not yet supported for large projects. Try with --json.',
-        );
+        if (!options.quiet) {
+          console.log(
+            '--print-deps option not yet supported for large projects. Try with --json.',
```
Comment (anthogez) [inline src/lib/print-deps.ts:34]: Please add some tests to validate this alias ^^

File: src/cli/args.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ const alias = abbrev(
 );
 alias.d = 'debug'; // always make `-d` debug
 alias.t = 'test';
+alias.q = 'quiet';
```
Comment (JackuB) [inline src/cli/args.ts:29]: Welp, our docs says we already support it https://support.snyk.io/hc/en-us/articles/360003812578-CLI-reference 🤪

> -q, --quiet Silence all output.

Then this should disable stdout altogether?

Comment (JackuB) [reply src/cli/args.ts:29]: Screenshot 2020-11-08 at 12 19 29

Yep

Comment (aviadhahami) [reply src/cli/args.ts:29]: I need to enable stdout, but disable specific error logs.
Maybe `--silence-errors`?

**Comment by JackuB** [comment]
> A classic example will be the print-deps warning for large graphs/trees;

> `--print-deps option not yet supported for large projects. Try with --json.`

Why do we treat CLI _not doing something I asked it to do_ as a warning that I want to silence? Isn't it a failure then? 🤔 Or if it's a case where a log message and not doing something is sensible, then why not fallback to `--json` as well, automatically?

**Comment by aviadhahami** [comment]
Closing as we already support it

**Pull request closed without merging**